src/redux/explorer.js

```diff
@@ -292,6 +308,12 @@ export const explorerInit = () => async (dispatch, getState) => {
   }
 };
 
+export const emitPortfolioRequest = (address, currency) => getState => {
+  const nativeCurrency = currency || getState().settings;
```

Comment:
I did this on purpose because we need to track prices in USD for consistency purpose.
For now, the only place where this is called, it's passing 'usd' as a param, but I wanted to make it flexible in case we want to use this for user facing info, which based on my convo with Christian, we will.

Reply:
oh yeah, that's fine - what i mean is it will literally be equal to all of getState().settings not getState().settings.nativeCurrency

Reply:
☠️